legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-dbExtension/legend-engine-xt-relationalStore-databricks/legend-engine-xt-relationalStore-databricks-grammar/src/test/java/org/finos/legend/engine/language/pure/grammar/test/TestGrammarExtensionsAvailable.java

```diff
@@ -0,0 +1,64 @@
+// Copyright 2023 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.engine.language.pure.grammar.test;
+
+import org.eclipse.collections.api.list.MutableList;
+import org.eclipse.collections.impl.factory.Lists;
+import org.finos.legend.engine.language.pure.compiler.toPureGraph.DatabricksCompilerExtension;
+import org.finos.legend.engine.language.pure.compiler.toPureGraph.extension.CompilerExtension;
+import org.finos.legend.engine.language.pure.grammar.from.DatabricksGrammarParserExtension;
+import org.finos.legend.engine.language.pure.grammar.from.IRelationalGrammarParserExtension;
+import org.finos.legend.engine.language.pure.grammar.to.DatabricksGrammarComposerExtension;
+import org.finos.legend.engine.language.pure.grammar.to.extension.PureGrammarComposerExtension;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.ServiceLoader;
+
+public class TestGrammarExtensionsAvailable
```

Comment:
Havind dbtype in class names will help in code navigation. Else too many duplicates